src/Leaflet/Core/Types.purs

```diff
@@ -132,13 +154,13 @@ type PopupConf =
 
 type IconConf =
   InteractiveLayerConf
-  ( iconUrl ∷ URIRef
-  , iconRetinaUrl ∷ URIRef
+  ( iconUrl ∷ LeafURIRef

```

Comment:
`iconUrl`, `iconRetinaUrl`, `shadowUrl` and `shadowRetinaUrl` are common uris btw, they don't have any special meanings for `{{x|y|z}}`

Reply:
I don't understand what you mean, can you enplane?

Reply:
`setURI` function sets special uri that could have `{` and `}` inside. `iconUrl` and friends couldn't they are just old plain URI w/o any fancy template strings inside.